WeatherApp/BL/IWeatherHistoryService.cs

```diff
@@ -1,10 +1,13 @@
-﻿using System.Collections.Generic;
+﻿using BL.Models;
+using System;
 using System.Threading.Tasks;
 
 namespace BL
 {
     public  interface IWeatherHistoryService
     {
+        Task<WeatherHistory> GetWeatherHistory(string cityName, DateTime from, DateTime to);
```

Comment:
Try to use TimeRange instead of two DateTime parameters

Reply:
TimeRange parameter is now used.